gradle/libs.versions.toml

```diff
@@ -177,6 +178,10 @@ forkhandles-bom = { module = "dev.forkhandles:forkhandles-bom", version.ref = "f
 forkhandles-fabrikate4k = { module = "dev.forkhandles:fabrikate4k" }
 glide = { module = "com.github.bumptech.glide:glide", version.ref = "glide" }
 glide-compiler = { module = "com.github.bumptech.glide:compiler", version.ref = "glide" }
+http4k-bom = { module = "org.http4k:http4k-bom", version.ref = "http4kBom" }
+http4k-core = { module = "org.http4k:http4k-core" }
+http4k-client-okhttp = { module = "org.http4k:http4k-client-okhttp" }
+http4k-format-gson = { module = "org.http4k:http4k-format-gson" }
```

Comment:
I'd prefer it if we limited ourselves to one JSON library. We're already using Moshi and it also seems to be supported by http4k.

Reply:
Not a big deal, just by default it reads JSON arrays as `LinkedHashTreeMap` and forces the use of `Array`, resulting in the need to convert it to a `List` by hand. Not a big deal but long term I would suggest replacing Moshi with a pure Kotlin serializer.